scripts/zones/West_Sarutabaruta/npcs/Twinkle_Tree.lua

```diff
@@ -12,19 +12,31 @@ require("scripts/globals/quests")
 -----------------------------------
 local entity = {}
 
+local entity = {}
+
+local function isInTime()
+    return VanadielHour() <= 3
+end
+
+local function checkQuestStatus(player)
+    return player:getQuestStatus(xi.quest.log_id.WINDURST, xi.quest.id.windurst.TO_CATCH_A_FALLING_STAR)
+end
+
```

Comment:
a little unnecessary making these local functions.

Reply:
A few added lines to make them local functions makes it read cleaner and puts those multiple logic usages into a single usage location.

Reply:
It's been our style to not use functions if it's only used in one place.

Reply:
It's in our code style to not use functions that do such minimal work. Also let's be easy on resolving conversations if no reply has happened.

Reply:
Removed the isInTime function due to its simplicity.  Kept the checkQuestStatus to reduce the copy/paste of long property accessors and reduce potential errors.

Reply:
Again we wouldn't make a function that performs what a single line would complete. I get where you're coming from, but because we're using multi-line if then statements, this shouldn't affect anything. (I'm just trying to maintain code styling here, not judgement in saying who's right or wrong)

Reply:
Should be all set now.